tensorflow/core/kernels/mkl/mkl_batch_matmul_op.cc

```diff
@@ -135,8 +135,9 @@ class BatchMatMulMkl : public OpKernel {
 
     // Compute parameters for DNNL matmul primitive.
     MklBatchMatMulHelper bmm;
-    auto params = bmm.CreateMatMulParams(lhs.shape(), rhs.shape(), out_shape,
-                                         adj_x_, adj_y_);
+    string prefix = "batchmatmul";
+    auto params = bmm.CreateMatMulParams(prefix, lhs.shape(), rhs.shape(),
+                                         out_shape, adj_x_, adj_y_);
```

Comment:
Nit: Since we don't use this string anywhere else, I think we can pass it directly to the function
```suggestion
    auto params = bmm.CreateMatMulParams("batchmatmul", lhs.shape(), rhs.shape(),
                                         out_shape, adj_x_, adj_y_);
```
Or
```suggestion
    auto params = bmm.CreateMatMulParams(/*prefix=*/"batchmatmul", lhs.shape(), rhs.shape(),
                                         out_shape, adj_x_, adj_y_);
```
(Please reformat the file if the line goes over 80 characters.)

Reply:
The reason I used prefix is because the function bmm.CreateMatMulParams() interface needs a std::string reference for the first parameter. if we pass "batchmatmul", which is a const char array.  we will have compilation error. since I don't want to change the interface of CreateMatMulParams(), so i just construct the string prefix with "batchmatmul".

Reply:
Makes sense. Thank you for the clarifications!